GIT_CONFLICTS.md

```diff
@@ -0,0 +1,98 @@
+## Common QnA
+
+<details>
+<summary>
+<em>How can I know if I’m in a bad situation?</em>
+</summary>
+
+- Run `gitk` or `gitk --all` to visualize the commit tree.
+- Check if `git log` shows any merge commits, indicating a lack of proper synchronization with the upstream branch.
+</details>
+
+<details>
+<summary>
+<em>How to Hard Sync Master with Upstream</em>
+</summary>
+
+To hard sync your local master branch with the upstream master branch, follow these steps in your main branch:
+
+- `git remote add upstream /url-to-original-repo`
+- `git fetch upstream`

```

Comment:
I can say it directly taken from the AI and not tested, how can `/url-to-original-rep` work? 
Acceptance criteria was everthing should be tested and working, and for now only 2 points we need to have as previous review comment